### PR TITLE
Help for install command

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by Fatme** [comment]
@ikoevska

File: docs/man_pages/project/creation/install.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+General | `$ tns install [<Path To Package.json File>] [--path] [--copy-from]`
+
+Creates and prepares a NativeScript project from the specified package.json file using the default template or from an existing NativeScript project.
+
+### Options
+* `--path` - Specifies the directory where you want to create the project, if different from the current directory. The directory must be empty.
+* `--copy-from` - Specifies a directory which contains an existing NativeScript project. If not set, the NativeScript CLI creates the project from the default hello-world template.
+
+### Attributes
+* `<Path To Package.json File>` is the path to package.json file or directory that contains package.json file. <% if(isConsole) { %>For more information about the `package.json` specification, see https://github.com/NativeScript/nativescript-cli/blob/master/PLUGINS.md#packagejson-specification<% } %>. If the path to package.json file is not specified, NativeScript CLI searches for package.json in current directory.
```
Comment (teobugslayer) [inline docs/man_pages/project/creation/install.md]: The link is to our plugin specification, not to package.json one.

File: docs/man_pages/project/creation/install.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+install
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns install [<Path To Package.json File>] [--path] [--copy-from]`
+
+Creates and prepares a NativeScript project from the specified package.json file using the default template or from an existing NativeScript project.
```
Comment (ikoevska) [inline docs/man_pages/project/creation/install.md]:
```
Creates and prepares a NativeScript project from one of the following sources:
* a `package.json` file
* an existing project
```

File: docs/man_pages/project/creation/install.md (hunk 1 above)
Comment (ikoevska) [inline docs/man_pages/project/creation/install.md]:
```
`package.json`
```

Comment (ikoevska) [inline docs/man_pages/project/creation/install.md]:
```
if the path to `package.json` is not specified, the NativeScript CLI searches for `package.json` in the current directory.
```

**Comment by ikoevska** [comment]
We need the Related Commands section
Also, this needs to be added in index.md and in the Related Commands sections for the other commands in this directory (e.g. create)

File: docs/man_pages/project/configuration/install.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+install
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns install [--path]`
+
+Installs all platforms and dependencies described in package.json file.
```
Comment (ikoevska) [inline docs/man_pages/project/configuration/install.md]: the `package.json` file in the current directory.

Comment (ikoevska) [inline docs/man_pages/project/configuration/install.md]: Apply this comment to the descriptions of the command that appear in index.md and the affected related commands.

**Comment by ikoevska** [comment]
:+1:

File: docs/man_pages/project/configuration/install.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+install
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns install [--path]`
+
+Installs all platforms and dependencies described in the `package.json` file in the current directory.
```
Comment (ikoevska) [inline docs/man_pages/project/configuration/install.md]: Forgot the 

```<% if(isHtml) { %> The `package.json` file must be a valid `package.json` describing the configuration of a NativeScript project. If missing or corrupted, you can recreate the file by running `tns init` in the directory of a NativeScript project. <% } %>
``` 
part :)

**Pull request merged**